src/VacanciesManage/SFA.DAS.VacanciesManage/Application/Recruit/Commands/CreateVacancy/CreateVacancyCommandHandler.cs

```diff
@@ -75,5 +97,25 @@ public async Task<CreateVacancyCommandResponse> Handle(CreateVacancyCommand requ
                 VacancyReference = result.Body.ToString()
             };
         }
+
+        /// <summary>
+        /// Method to check if the given ukprn number is a valid training provider with Main or Employer Profile with Status not equal to "Not Currently Starting New Apprentices".
+        /// </summary>
+        /// <param name="ukprn">ukprn number.</param>
+        /// <returns>boolean.</returns>
+        private async Task<bool> IsTrainingProviderMainOrEmployerProfile(int ukprn)
+        {
+            var provider = await _roatpCourseManagementApiClient.Get<GetProvidersListItem>(new GetProviderRequest(ukprn));
+
+            _logger.LogDebug($"Response from Outer API: Provider:{JsonConvert.SerializeObject(provider)}");
+            _logger.LogCritical($"Response from Outer API: Provider:{JsonConvert.SerializeObject(provider)}");
+            _logger.LogError($"Response from Outer API: Provider:{JsonConvert.SerializeObject(provider)}");
```

Comment:
Can remove these

Reply:
obsolete code removed as not required added it for troubleshooting, but has been removed from the codebase.